core/src/OpenACC/Kokkos_OpenACC_ParallelReduce_Range.hpp

```diff
@@ -126,12 +124,13 @@ namespace Kokkos::Experimental::Impl {
 
 template <class IndexType, class ValueType, class Functor>
 void OpenACCParallelReduceSum(Schedule<Static>, int chunk_size, IndexType begin,
-                              IndexType end, ValueType& val,
+                              IndexType end, ValueType& aval,
                               Functor const& afunctor, int async_arg) {
   // FIXME_OPENACC FIXME_NVHPC workaround compiler bug (incorrect scope
   // analysis)
   // NVC++-S-1067-Cannot determine bounds for array - functor
   auto const functor(afunctor);
+  auto val = aval;
```

Comment:
Why did you change this?

Reply:
Local scalar variables are much easier for an OpenACC compiler to analyze their sharing rules and optimize.

Reply:
Please open another PR for these changes

Reply:
Moved to #5696